### PR TITLE
Navigating to error line number in Rider from Godot editor debugger console is off-by-one.

### LINKED ISSUE #61: Navigating to error line number in Rider from Godot editor debugger console is off-by-one.
The error line number in the Godot editor debugger console that is referencing an attached script in Rider in the stack trace is correct, but double clicking the specific stack trace item containing a file name and line number (.i.e., ScriptName.cs:LineNumber @ method signature) navigates to the error line minus one in Rider.

### PR DESCRIPTION
**Pull request by van800**
Fix https://github.com/JetBrains/godot-support/issues/61

### CODE REVIEW

File: modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs [hunk 1]
```diff
@@ -104,7 +104,7 @@ public static void OpenFile(string slnPath, string scriptPath, int line)
             if (line >= 0)
             {
                 args.Add("--line");
-                args.Add(line.ToString());
+                args.Add((line+1).ToString()); // https://github.com/JetBrains/godot-support/issues/61
```
Comment (aaronfranke) [inline modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs:107]:
```suggestion
                args.Add((line + 1).ToString()); // https://github.com/JetBrains/godot-support/issues/61
```

Per the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html), spaces around operators.

Comment (van800) [reply modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs]: Thanks for noticing. Fixed

**Comment by akien-mga** [comment]
It would be nice to amend the commit message to be more explicit about what the commit does (i.e. in English and not just with a URL), e.g.:
```
Fix off by one error navigating to line number in Rider

Fixes https://github.com/JetBrains/godot-support/issues/61
```

This makes the commit message useful when not viewing it on GitHub.

**Comment by van800** [comment]
> It would be nice to amend the commit message to be more explicit about what the commit does (i.e. in English and not just with a URL), e.g.:
> 
> ```
> Fix off by one error navigating to line number in Rider
> 
> Fixes https://github.com/JetBrains/godot-support/issues/61
> ```
> 
> This makes the commit message useful when not viewing it on GitHub.

done

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**